src/ripple/app/tests/Regression_test.cpp

```diff
@@ -157,11 +159,58 @@ struct Regression_test : public beast::unit_test::suite
         test256r1key (becky);
     }
 
+    static
+    std::unique_ptr<Config>
+    makeConfig(std::map<std::string, std::string> extra = {})
```

Comment:
If possible please make this a lambda inside the function that uses it, so we can add more tests without stepping on identifier names.